{"name": "Проверка заказа в работе", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_feed_page.TestFeedPage object at 0x1068b82c0>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"0e181bc2-3bb3-4afa-b6c2-1da4b9f27404\")>, set_user_tokens = None\n\n    @pytest.mark.parametrize('driver', ['chrome', 'firefox'], indirect=True)\n    @allure.title('Проверка заказа в работе')\n    @allure.description('Проверяем, что после оформления заказа его номер появляется в разделе \"В работе\"')\n    def test_new_order_appears_in_feed_progress_section(self, driver, set_user_tokens):\n        driver.get(Urls.MAIN_URL)\n        main_page = MainPage(driver)\n        feed_page = FeedPage(driver)\n        main_page.wait_for_order_section()\n        main_page.drag_ingredient_to_order_section()\n        main_page.click_order_button()\n>       main_page.wait_for_new_order_modal_disapear()\n\ntests/test_feed_page.py:95: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages/main_page.py:50: in wait_for_new_order_modal_disapear\n    self.check_element_is_invisible(locator)\npages/base_page.py:35: in check_element_is_invisible\n    WebDriverWait(self.driver, 10).until_not(ec.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0e181bc2-3bb3-4afa-b6c2-1da4b9f27404\")>, method = <function presence_of_element_located.<locals>._predicate at 0x106d27060>, message = ''\n\n    def until_not(self, method: Callable[[D], T], message: str = \"\") -> Union[T, Literal[True]]:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value evaluates to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`, or\n                  ``True`` if `method` has raised one of the ignored exceptions\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if not value:\n                    return value\n            except self._ignored_exceptions:\n                return True\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nvenv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:128: TimeoutException"}, "description": "Проверяем, что после оформления заказа его номер появляется в разделе \"В работе\"", "steps": [{"name": "Ожидаем появления секции конструктора заказов", "status": "passed", "start": 1713085649407, "stop": 1713085649419}, {"name": "Перетаскиваем булочку в секцию заказа", "status": "passed", "start": 1713085649419, "stop": 1713085650495}, {"name": "Кликам по кнопке оформления заказа", "status": "passed", "start": 1713085650495, "stop": 1713085650715}, {"name": "Ожидаем получения номера нового заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"/Users/radiusme/coding/Diplom_3/venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/Users/radiusme/coding/Diplom_3/pages/main_page.py\", line 50, in wait_for_new_order_modal_disapear\n    self.check_element_is_invisible(locator)\n  File \"/Users/radiusme/coding/Diplom_3/pages/base_page.py\", line 35, in check_element_is_invisible\n    WebDriverWait(self.driver, 10).until_not(ec.presence_of_element_located(locator))\n  File \"/Users/radiusme/coding/Diplom_3/venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 128, in until_not\n    raise TimeoutException(message)\n"}, "start": 1713085650715, "stop": 1713085660873}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1713085649270, "stop": 1713085660876, "uuid": "545e14f8-72c8-481b-88fd-086511167e8a", "historyId": "7e7e5dc0fff02327c3ea19d23bcab07d", "testCaseId": "db70f6bfc931c67b8fe880194db65d0f", "fullName": "tests.test_feed_page.TestFeedPage#test_new_order_appears_in_feed_progress_section", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_feed_page"}, {"name": "subSuite", "value": "TestFeedPage"}, {"name": "host", "value": "Radiuss-MacBook-Air.local"}, {"name": "thread", "value": "70542-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_feed_page"}]}